tests/Header.test.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { render, screen } from '@testing-library/react';
+import Header from '../src/ui/Header';
+import { Navigator, ReaderState } from '../src/types';
+
+test('render header bar', () => {
+  const readerState = {} as ReaderState;
+  const navigator = {} as Navigator;
+
+  render(<Header readerState={readerState} navigator={navigator} />);
+
+  // Should also test the href once everthings been implemented in the furture
+  // By default it should show the NYPL logo and a link
+  expect(
+    screen.getByRole('link', { name: 'Return to Digital Research Books' })
```

Comment:
Let's make this default "Return to NYPL" instead of "Digital Research Books" and make the href `www.nypl.org`